### PR TITLE
feat: add block storage onboarding

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

eb9cfc32 — feat: add block storage onboarding

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<pci-project-empty guides="$ctrl.guides">
+    <p data-translate="pci_projects_project_storages_blocks_onboarding_content1"></p>
+    <strong data-translate="pci_projects_project_storages_blocks_onboarding_content2"></strong>
+    <p data-translate="pci_projects_project_storages_blocks_onboarding_content3"></p>
+    <p data-translate="pci_projects_project_storages_blocks_onboarding_content4"></p>
+
+    <oui-button variant="primary" text="Primary"
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.html]: Must be in a translation file.

File: packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.routing.js [hunk 2]
```diff
@@ -3,6 +3,13 @@ export default /* @ngInject */ ($stateProvider) => {
     .state('pci.projects.project.storages.blocks', {
       url: '/blocks?help',
       component: 'pciProjectStorageBlocks',
+
+      redirectTo: (trans) => {
+        const resolvePromise = trans.injector().getAsync('storages');
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.routing.js]: Any reason not to 
```suggestion
        return trans.injector()
          .getAsync('storages')
          .then(...)
```
?

File: packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import 'oclazyload';
+
+const moduleName = 'ovhManagerPciStoragesBlocksOnboardingLazyLoading';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+    'oc.lazyLoad',
+  ])
+  .config(/* @ngInject */($stateProvider) => {
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/index.js]:
```suggestion
  .config(/* @ngInject */ ($stateProvider) => {
```
😅

File: packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.constants.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export const GUIDES = [
+  {
+    id: 'create-attach-volume',
+    link: 'https://docs.ovh.com/gb/en/public-cloud/create-an-additional-volume-and-attach-it-to-an-instance/',
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.constants.js]: Only the english guides for now ?

File: packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.html (hunk 1 above)
Comment (FredericEspiau) [reply packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.html]: `data-`

File: packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  "pci_projects_project_storages_blocks_onboarding_content3": "Ajoutez facilement de l'espace disque à vos instances avec ces volumes. Ceux-ci peuvent être agrandis selon vos besoins et vous avez aussi la possibilité d’en attacher plusieurs par instance. Ce lien s’effectue à chaud ; vous déplacez ensuite un volume d'une instance à une autre dynamiquement dans la même localisation.",
+  "pci_projects_project_storages_blocks_onboarding_content4": "Choisissez la taille qui vous convient, de 10 Go jusqu'à 4 To. Les volumes sont également disponibles en deux versions : classic, pour privilégier le coût au Go, et high-speed, afin de bénéficier des meilleures performances.",
+  "pci_projects_project_storages_blocks_onboarding_action_label": "Créer un volume",
+
+  "pci_projects_project_storages_blocks_onboarding_guides_create-attach-volume_title": "Créer un volume supplementaire et l’attacher a une instance",
+  "pci_projects_project_storages_blocks_onboarding_guides_create-attach-volume_description": "Documentation pas a pas traitant de la creation et de l’attachement d’un disque additionnel a un serveur cloud.",
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/translations/Messages_fr_FR.json]:
```suggestion
  "pci_projects_project_storages_blocks_onboarding_guides_create-attach-volume_description": "Documentation pas à pas traitant de la creation et de l’attachement d’un disque additionnel a un serveur cloud.",
```

File: packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  "pci_projects_project_storages_blocks_onboarding_content2": "Disposez de volumes de stockage sécurisés, grâce à une triple réplication des données, et utilisables comme des disques supplémentaires.",
+  "pci_projects_project_storages_blocks_onboarding_content3": "Ajoutez facilement de l'espace disque à vos instances avec ces volumes. Ceux-ci peuvent être agrandis selon vos besoins et vous avez aussi la possibilité d’en attacher plusieurs par instance. Ce lien s’effectue à chaud ; vous déplacez ensuite un volume d'une instance à une autre dynamiquement dans la même localisation.",
+  "pci_projects_project_storages_blocks_onboarding_content4": "Choisissez la taille qui vous convient, de 10 Go jusqu'à 4 To. Les volumes sont également disponibles en deux versions : classic, pour privilégier le coût au Go, et high-speed, afin de bénéficier des meilleures performances.",
+  "pci_projects_project_storages_blocks_onboarding_action_label": "Créer un volume",
+
+  "pci_projects_project_storages_blocks_onboarding_guides_create-attach-volume_title": "Créer un volume supplementaire et l’attacher a une instance",
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/translations/Messages_fr_FR.json]:
```suggestion
  "pci_projects_project_storages_blocks_onboarding_guides_create-attach-volume_title": "Créer un volume supplementaire et l’attacher à une instance",
```

File: packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/translations/Messages_fr_FR.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
+{
+  "pci_projects_project_storages_blocks_onboarding_content1": "Vous n'avez pas encore créé de service de Block Storage.",
+  "pci_projects_project_storages_blocks_onboarding_content2": "Disposez de volumes de stockage sécurisés, grâce à une triple réplication des données, et utilisables comme des disques supplémentaires.",
+  "pci_projects_project_storages_blocks_onboarding_content3": "Ajoutez facilement de l'espace disque à vos instances avec ces volumes. Ceux-ci peuvent être agrandis selon vos besoins et vous avez aussi la possibilité d’en attacher plusieurs par instance. Ce lien s’effectue à chaud ; vous déplacez ensuite un volume d'une instance à une autre dynamiquement dans la même localisation.",
+  "pci_projects_project_storages_blocks_onboarding_content4": "Choisissez la taille qui vous convient, de 10 Go jusqu'à 4 To. Les volumes sont également disponibles en deux versions : classic, pour privilégier le coût au Go, et high-speed, afin de bénéficier des meilleures performances.",
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/translations/Messages_fr_FR.json]:
```suggestion
  "pci_projects_project_storages_blocks_onboarding_content4": "Choisissez la taille qui vous convient, de 10 Go jusqu'à 4 To. Les volumes sont également disponibles en deux versions : « classic », pour privilégier le coût au Go, et « high-speed », afin de bénéficier des meilleures performances.",
```

File: packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.html (hunk 1 above)
Comment (cbourgois) [reply packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.html]: Nope : 
* `text` is deprecated but `ovh-ui-angular` documentation is out of date.
* we use the transclude value, so `text="Primary"` should be removed...

Comment (antleblanc) [reply packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.html]: So you can get rid of.

File: packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.constants.js (hunk 4 above)
Comment (cbourgois) [reply packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.constants.js]: Nice catch ;)

Comment (cbourgois) [reply packages/manager/modules/pci/src/projects/project/storages/blocks/onboarding/onboarding.constants.js]: English guides only for now.

**Pull request merged**